### ISSUE TITLE
scm_new_port_table_entry has been removed in Guile-2.2+

### ISSUE THREAD

**Issue by NalaGinrut** [spk121/zile#2]
Unfortunately, you can't see it in the commit log on the web, since it's skipped by a special sign. But you can still see it in the 'git log' command.
This breaks the compilation of zile-on-guile

```
commit 9417fdb80fb5db4f657c9a329faaa61162ab996b
Author: Ludovic Courtès <ludo@gnu.org>
Date:   Fri Feb 16 14:26:56 2018 +0100

    doc: Fix typo about SRFI-4 syntax.
    
    Reported by Matt Wette <matt.wette@gmail.com>.
...skipping...
      (scm_new_port_table_entry): Remove; not useful.
```

**Comment by spk121** [comment]
Confirmed.
Thanks for the report.

**Comment by spk121** [comment]
Patched in b11eb3d0b1e4de6cb53cda2d961641d84b3712dc
Will appear in forthcoming release
